client/my-sites/stats/site.jsx

```diff
@@ -536,6 +599,18 @@ function StatsBody( { siteId, chartTab = 'views', date, context, isInternal, ...
 					className="stats__section-header modernized-header"
 					title={ isOdysseyStats ? STATS_PRODUCT_NAME : STATS_PRODUCT_NAME_IMPR }
 					titleLogo={ isOdysseyStats ? <JetpackLogo size={ 24 } monochrome={ false } /> : null }
+					rightSection={
+						shouldRenderModuleToggler && (
+							<PageModuleToggler
+								availableModuleToggles={ availableModuleToggles }
```

Comment:
It feels like the better place to get most of those props would be within `PageModuleToggler` itself. You should still be able to isolate a simple visual layer, and then wrap it up and export with logics.

Otherwise the Traffic would become more bloated.

Reply:
@dognose24, @kangzj - When we move these props inside of `PageModuleToggler`, I am wondering if [props related to the tooltip can be moved too](https://github.com/Automattic/wp-calypso/blob/07272834b5cbf0e2a0d51ff29cb9fe4ccc367b23/client/my-sites/stats/site.jsx#L536-L543)? 

The `PageModuleToggler` seems bit generic and not tied to any page, But this is also not used anywhere else at the moment - So I am thinking we could move these props too inside the `PageModuleToggler` let that component manage hide/show of the tooltip (along with all other props)

Reply:
The [refactor I mentioned](https://github.com/Automattic/wp-calypso/pull/102881#discussion_r2063904079) involved moving those functions inside `PageModuleToggler` and supporting the passing of a page name, e.g., `traffic`, to enable [those actions](https://github.com/Automattic/wp-calypso/blob/07272834b5cbf0e2a0d51ff29cb9fe4ccc367b23/client/my-sites/stats/site.jsx#L538-L561) to work for specific page modules.